nextmv/options.py

```diff
@@ -221,9 +360,7 @@ def _parameter_value(parameter: Parameter, value: Any) -> Any:
 
         value = str(value).lower()
 
-        if value in ("true", "1", "t", "y", "yes"):
+        if value in ("true", "True", "TRUE", "1", "t", "T", "y", "Y", "yes", "Yes", "YES"):
```

Comment:
The different cases are not necessary. `value = str(value).lower()` will take care of that. You can eliminate any check for uppercase.

```suggestion
        if value in ("true", "1", "t", "y", "yes"):
```

Reply:
Good catch! Thank you!